src/main.rs

```diff
@@ -1,6 +1,4 @@
 #![feature(lang_items)]
```

Comment:
If you compile with panic=abort then `eh_personality` is not necessary and this feature gate can be removed.

Reply:
Nice! I think only a stable way to build `core` is missing now, then we could compile this crate on stable Rust :tada:.